### DISCUSSION TITLE
only_modifications_with_file_types is not working

### DISCUSSION THREAD

**Discussion by AnkushPrasad05** [ishepard/pydriller#219] [Q&A]
I was trying to get files from the latest commit with specific file extension for which I used "only_modifications_with_file_types" which is giving me all the files from the latest commit not the files with specific extension. Please help.

Please find below code snippet:

> commit = list(Repository(self.root_directory,
>                                      only_in_branch=str(repo.active_branch.name),
>                                      from_commit=repo.git.rev_parse("HEAD"),
>                                      only_modifications_with_file_types=['.json']).traverse_commits())[0]

**Comment by stefanodallapalma** [comment]
Hi @AnkushPrasad05, could you please provide a reproducible example?
